documentation/Command-Line-Arguments.md

```diff
@@ -11,9 +11,24 @@ Note that if you absolutely need control over the command line argument processi
 Trapperkeeper supports three command-line arguments:
 
 * `--config/-c`: The path to the configuration file or directory. This option is used to initialize the configuration service. This argument is optional; if not specified, Trapperkeeper will act as if you had given it an empty configuration file.
```

Comment:
Unrelated to this PR since this functionality was pre-existing, but might be good to update this line while we're in here to indicate that a comma separated list of files / directories can be provided for this option also.